dev/com.ibm.ws.kernel.reporting_fat/fat/src/com/ibm/ws/kernel/reporting/fat/CVEDataTest.java

```diff
@@ -0,0 +1,130 @@
+package com.ibm.ws.kernel.reporting.fat;
```

Comment:
Similarly the package name should start with io.openliberty.  You should also consider the project starting with io.oepnliberty as well, but I understand if you want to keep it by the other projects that start with com.ibm.ws.kernel projects.

Reply:
Done.